src/App.vue

```diff
@@ -1,17 +1,53 @@
 <template>
   <div id="app">
-    <h1>Test App</h1>
-    <button>Go</button>
+    <h1>Test app</h1>
+    <button @click="getPosts()">Go</button>
+    <table>
```

Comment:
Since table is another component. I think we should create a new component for table and import in our` App.vue` file.

Reply:
Refactor the code accordingly